src/build.rs

```diff
@@ -1186,6 +1186,10 @@ pub fn build(filter: &Option<regex::Regex>, path: &str, no_timing: bool) -> Resu
             .as_secs_f64()
     );
 
+    if !package_tree::valide_package_dependencies(&packages) {
```

Comment:
I'm not sure if we should stop the whole build process after this validation.
Also, I'm not sure if I'm stopping it in the best way 🤔